### DISCUSSION TITLE
Can't use Enchanted Sundial

### DISCUSSION THREAD

**Discussion by hdseventh** [Pryaxis/TShock#2431] [Q&A] [answered]
I can't use Enchanted Sundial even with SuperAdmin permissions, is this a bug or is there any config or command  i should use?
Checked the ForceTime config in config file, its set to "normal". I also tried to add the permission to use sundial to the default group, but still can't use it when im using a default group character, it shows no "You don't have the permission" message either.

Tested with fresh TShock 4.5.5

**Comment by Junii23** [comment]
I had this same issue. So far the only way I've found to fix this was to downgrade to TShock 4.5.4. If you don't need the new stuff in 4.5.5 I would suggest try downgrading and seeing if it fixes it for you.

**Comment by csmir** [comment]
> I can't use Enchanted Sundial even with SuperAdmin permissions, is this a bug or is there any config or command i should use?
> Checked the ForceTime config in config file, its set to "normal". I also tried to add the permission to use sundial to the default group, but still can't use it when im using a default group character, it shows no "You don't have the permission" message either.
> 
> Tested with fresh TShock 4.5.5

Moving to issue, will be looked at. I can confirm this is currently not working and that the code needs a review. 

No reject is sent in log; Nothing is sent to player. Permission has no effect.

**Comment by csmir** [comment, marked as answer]
A fix is upcoming for the next release. Until the next release is published, downgrading to 4.5.4 is suggested _only if sundial usage is absolutely required._

If use of this item is not required for gameplay, reverting to older versions is not suggested. It is not uncommon for new releases to change or overwrite certain dated features, which makes older versions unable to access them in rare cases.
